{"name": "Проверка ручки создания курьера с существующими параметрами (login, password)", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется.'\n  - Этот логин уже используется.\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <test_creating_courier.TestCreatingCourier object at 0x00000148F3E3F750>\n\n    @allure.title(        'Проверка ручки создания курьера с существующими параметрами (login, password)')\n    @allure.description('Сравниваем ожидаемый статус-код ответа \"409\" с фактическим')\n    def test_creating_courier_duplicate(self):\n            payload = {\n                \"login\": \"vorona\",\n                \"password\": \"987654321qwerasdzxc\",\n                \"firstName\": \"voronickhina\"\n            }\n            r = requests.post(f\"{url}/courier\", data=payload)\n    \n            assert r.status_code  == 409\n>           assert r.json()[\"message\"] == \"Этот логин уже используется.\"\nE           AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется.'\nE             - Этот логин уже используется.\nE             + Этот логин уже используется. Попробуйте другой.\n\ntest_creating_courier.py:56: AssertionError"}, "description": "Сравниваем ожидаемый статус-код ответа \"409\" с фактическим", "start": 1698872461080, "stop": 1698872461210, "uuid": "862a9f5c-a277-4e96-bd1d-9178e11eecc2", "historyId": "f95a78fd05683dcef84c8bebf22740ef", "testCaseId": "f95a78fd05683dcef84c8bebf22740ef", "fullName": "test_creating_courier.TestCreatingCourier#test_creating_courier_duplicate", "labels": [{"name": "suite", "value": "test_creating_courier"}, {"name": "subSuite", "value": "TestCreatingCourier"}, {"name": "host", "value": "WIN-48K729GVFIE"}, {"name": "thread", "value": "16156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_creating_courier"}]}